src/lib/stores/marketplace.ts

```diff
@@ -0,0 +1,686 @@
+                name: 'APPWRITE_COLLECTION_ID',
+                description: `The ID of the collection in the Appwrite database to sync. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/collections">Learn More</a>`,
+                placeholder: '7c3e8...2a9f1',
+                required: true
+            },
+            {
+                name: 'ALGOLIA_ADMIN_API_KEY',
+                description: `The admin API Key for your Algolia service. <a class="u-bold" target="_blank" href="https://www.algolia.com/doc/guides/security/api-keys/">Learn More</a>`,
+                placeholder: 'fd0aa...136a8',
+                required: true
+            },
+            {
+                name: 'ALGOLIA_INDEX_ID',
+                description: `The ID of the index in Algolia where the documents are to be synced. <a class="u-bold" target="_blank" href="https://www.algolia.com/doc/api-client/methods/indexing/">Learn More</a>`,
+                placeholder: 'appwrite_index',
+                required: true
+            },
+            {
+                name: 'ALGOLIA_SEARCH_API_KEY',
+                description: `The search API Key for your Algolia service. This key is used for searching the synced index. <a class="u-bold" target="_blank" href="https://www.algolia.com/doc/guides/security/api-keys/">Learn More</a>`,
+                placeholder: 'bf2f5...df733',
+                required: true
+            },
+            {
+                name: 'APPWRITE_ENDPOINT',
+                description: `The URL endpoint of the Appwrite server. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/getting-started-for-server">Learn More</a>`,
+                value: 'https://cloud.appwrite.io/v1',
+                placeholder: 'https://cloud.appwrite.io/v1',
+                required: false
+            }
+        ]
+    },
+    {
+        icon: 'icon-meilisearch',
+        id: 'sync-with-meilisearch',
+        name: 'Sync with Meilisearch',
+        tagline: 'Intuitive search bar for any data in Appwrite Databases.',
+        permissions: ['any'],
+        events: [],
+        cron: '',
+        timeout: 15,
+        runtimes: [
+            {
+                name: 'node-18.0',
+                commands: 'npm install',
+                entrypoint: 'src/main.js',
+                providerRootDirectory: 'node/sync-with-meilisearch'
+            }
+        ],
+        instructions: `For documentation and instructions check out [README.md](https://github.com/appwrite/templates/tree/main/node/sync-with-meilisearch) file.`,
+        vcsProvider: 'github',
+        providerRepositoryId: 'templates',
+        providerOwner: 'appwrite',
+        providerBranch: 'main',
+        variables: [
+            {
+                name: 'APPWRITE_API_KEY',
+                description: `The API Key to talk to Appwrite backend APIs. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/getting-started-for-server">Learn More</a>`,
```

Comment:
Please fix the `Learn more`, needs to be sentence case.

Also missing periods in every description after `Learn more</a>.